### DISCUSSION TITLE
Voice Playback during 2D replay

### DISCUSSION THREAD

**Discussion by lun7n** [akiver/cs-demo-manager#877] [Ideas]
Hello Dev,
Please make it possible to have voice comms during 2D playback if possible. Seeing player's comms and rotations based off of comms will add another dimension of analysis to 2D playback.

Specifically for faceit pug demos where this data is present for both teams.

**Comment by akiver** [comment]
Available since v3.14.0, see the [documentation](https://cs-demo-manager.com/docs/guides/2d-viewer)
